javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,45 @@
+var width = 420;
+var height = 70;
+var area = width * height;
+
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+var bill = 1073;
+var payment = 29400;
+var change = payment - bill;
+
+console.log('value of change:', change);
+console.log('typeof change:', typeof change);
+
+var quizzes = 100;
+var midterms = 4;
+var final = 20;
+var grade = ((quizzes + midterms + final) / 3);
+
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+
+var firstName = 'Lamberto';
+var lastName = 'Martinez';
+var fullName = firstName + ' ' + lastName;
+
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+
+var pH = 12.0;
+var isAcidic = pH < 7;
+
+console.log('value of is acidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+var headCount = 420;
+var isSparta = headCount === 300;
+
+console.log('value of isSparta:', isSparta);
+console.log('type of isSparta:', isSparta);
+
+var motto = fullName += ' is the GOAT';
```

Comment:
By doing `fullName += ' is the GOAT';` you're reassigning this string to fullName, which is not want you want to do, you want this to be assigned to motto. So your code should look like this

`var motto = fullName;`
`motto += ' is the GOAT';`